modAionImpl/test/org/aion/zero/impl/sync/handler/ResponseBlocksHandlerTest.java

```diff
@@ -0,0 +1,147 @@
+package org.aion.zero.impl.sync.handler;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import org.aion.p2p.IP2pMgr;
+import org.aion.p2p.Ver;
+import org.aion.p2p.impl1.P2pMgr;
+import org.aion.rlp.RLP;
+import org.aion.util.TestResources;
+import org.aion.zero.impl.sync.Act;
+import org.aion.zero.impl.sync.FastSyncManager;
+import org.aion.zero.impl.sync.msg.ResponseBlocks;
+import org.junit.Test;
+import org.slf4j.Logger;
+
+/**
+ * Unit tests for {@link ResponseBlocksHandler}.
+ *
+ * @author Alexandra Roatis
+ */
+public class ResponseBlocksHandlerTest {
+    private static final int peerId = Integer.MAX_VALUE;
+    private static final String displayId = "abcdef";
+
+    @Test
+    public void testHeader() {
+        Logger log = mock(Logger.class);
+        FastSyncManager fastSyncManager = mock(FastSyncManager.class);
+        IP2pMgr p2p = mock(P2pMgr.class);
+
+        ResponseBlocksHandler handler = new ResponseBlocksHandler(log, fastSyncManager, p2p);
+        // check handler header
+        assertThat(handler.getHeader().getVer()).isEqualTo(Ver.V1);
+        assertThat(handler.getHeader().getAction()).isEqualTo(Act.RESPONSE_BLOCKS);
+    }
+
+    @Test
+    public void testReceive_nullMessage() {
+        Logger log = mock(Logger.class);
```

Comment:
optional: could put this stuff in a `@Before` to save some typing

Reply:
I won't apply this suggestion now cause, given that the tests are already written, it creates addtional typing. I may apply it later if more tests are needed.